seed_data/management/commands/seed_db_test.py

```diff
@@ -140,6 +140,29 @@ class SeedDBDeserializationTests(MockedESTestCase):
         ]
     }
 
+    ELECTIVE_DATA = {
+        "title": "ELECTIVE",
+        "required_number": 2,
+        "courses": [
+            {
+                "title": "Digital Learning 400 (ELECTIVE)",
+                "position_in_program": 1,
+                "description": "An introductory course to Digital Learning",
+                "course_runs": [
```

Comment:
This seed data is not going to work, because you are requiring 2 courses from a set of 1 course.

Reply:
@annagav Feedback incorporated. Test data is updated to be more relatable, e.g. Created two elective courses for the set with 1 required value.